apps/vaults/contexts/useSolver.tsx

```diff
@@ -37,7 +37,7 @@ export const isSolverDisabled = {
 	[Solver.CHAIN_COIN]: false,
 	[Solver.INTERNAL_MIGRATION]: false,
 	[Solver.COWSWAP]: false,
-	[Solver.OPTIMISM_BOOSTER]: false,
+	[Solver.OPTIMISM_BOOSTER]: true,
```

Comment:
Thinking that these could be environment variables so that zaps can be disabled more effectively